plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -317,6 +317,9 @@ class ServicesTable extends React.Component {
       )
     });
 
+    const serviceIsDeleting =
```

Comment:
Please alsways add basic service methods to the service tree as well as we otherwise end up with plenty of these typchecks.